### PR TITLE
Issue6592 xc UI tests using china server

### PR DESCRIPTION
**Pull request by isabelrios**
It would be nice if we could have sync int tests using china server to realize when/if there are issues there. 

So far I can't find a way to use a testing account with that server, so let's check at least that we use that server and FxA page is loaded for it. 
This test will run as part of the sync int tests, daily on a Jenkins job

### CODE REVIEW

**Comment by isabelrios** [comment]
@nbhasin2 do you know if this is the correct site shown when using China server? 
Simulator Screen Shot - iPhone 8 - 2021-06-04 at 15 39 39

@bli-mozbj do you know if there is a verified testing account we could use for daily tests?

**Comment by nbhasin2** [comment]
@isabelrios That website seems correct. I know it doesn't look like our regular login website but this is the china server website.

**Comment by bli-mozbj** [comment]
>  
> @nbhasin2 do you know if this is the correct site shown when using China server?
 
Yes, it's the correct site.

> @bli-mozbj do you know if there is a verified testing account we could use for daily tests?

I do have some testing accounts, pls refer to https://docs.google.com/spreadsheets/d/1aXaNpnvNkR4QbQ0pv_tWf_nX0pgBxr_tfh741gjwUYM/edit?usp=sharing .

**Comment by isabelrios** [comment]
> > @nbhasin2 do you know if this is the correct site shown when using China server?
> 
> Yes, it's the correct site.
> 
> > @bli-mozbj do you know if there is a verified testing account we could use for daily tests?
> 
> I do have some testing accounts, pls refer to https://docs.google.com/spreadsheets/d/1aXaNpnvNkR4QbQ0pv_tWf_nX0pgBxr_tfh741gjwUYM/edit?usp=sharing .

thank you @bli-mozbj! yeah I can log in with those accounts but for the automated tests we will need them to be always verified for the simplicity and realiability of the tests. I will see if that's possible

**Comment by isabelrios** [comment]
Forgot to ask for review... we can start by adding just the check to the login page and then create the rest of tests to verify the sync with items in History/Bookmarks/Tabs/Logins

File: Shared/LaunchArguments.swift [hunk 1]
```diff
@@ -12,6 +12,7 @@ public struct LaunchArguments {
     public static let SkipETPCoverSheet = "FIREFOX_SKIP_ETP_COVER_SHEET"
     public static let ClearProfile = "FIREFOX_CLEAR_PROFILE"
     public static let StageServer = "FIREFOX_USE_STAGE_SERVER"
+    public static let ChinaServer = "FIREFOX_USE_CHINA_SERVER"
```
Comment (AaronMT) [inline Shared/LaunchArguments.swift:15]: Maybe `FxAChinaServer` (@nbhasin2 thoughts?)

**Review by AaronMT — APPROVED** [review]

File: Shared/LaunchArguments.swift (hunk 1 above)
Comment (nbhasin2) [reply Shared/LaunchArguments.swift:15]: `FxAChinaServer` sounds better

Comment (isabelrios) [reply Shared/LaunchArguments.swift:15]: Thanks I will modify that :)

**Pull request merged**